src/components/AnonymousChat/child/ChatItemMyTextV2.tsx

```diff
@@ -169,7 +169,7 @@ const ChatItemMyTextV2 = ({
 
   return (
     <View style={[styles.chatContainer, isContinuous ? {marginTop: dimen.normalizeDimen(-4)} : {}]}>
-      <ChatContextMenuView contextMenuType="MyChatContextMenu" chat={chatItem}>
+      <ChatContextMenuView contextMenuType="MyChatContextMenu" chat={chatItem} type={chatType}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change looks good. The `ChatContextMenuView` component now accepts a new prop `type` which is set to `chatType`. This allows the context menu to be aware of the chat type and adjust its behavior accordingly.
```diff
- <ChatContextMenuView contextMenuType="MyChatContextMenu" chat={chatItem}>
+ <ChatContextMenuView contextMenuType="MyChatContextMenu" chat={chatItem} type={chatType}>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->